.woodpecker.yml

```diff
@@ -76,17 +76,16 @@ steps:
     when:
       - event: pull_request
 
-  # TODO
-  # publish_to_crates_io:
-  #   image: *rust_image
-  #   commands:
-  #     - cargo install cargo-workspaces
-  #     - cp -r migrations crates/db_schema/
-  #     - cargo login "$CARGO_API_TOKEN"
-  #     - cargo workspaces publish --from-git --allow-dirty --no-verify --allow-branch "${CI_COMMIT_TAG}" --yes custom "${CI_COMMIT_TAG}"
-  #   secrets: [cargo_api_token]
-  #   when:
-  #     - event: tag
+  publish_to_crates_io:
+    image: *rust_image
+    commands:
+      - cargo install cargo-workspaces
+      - cp -r migrations crates/db_schema/
+      - cargo login "$CARGO_API_TOKEN"
+      - cargo workspaces publish --from-git --allow-dirty --no-verify --allow-branch "${CI_COMMIT_TAG}" --yes custom "${CI_COMMIT_TAG}"
```

Comment:
I don't think you need lines 82-84, and probably don't need cargo workspaces either.